releasenotes/notes/add-deprecation-warning-context-relevance-937df7e807ac1a8d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    "The output of the ContextRelevanceEvaluator will change in Haystack 2.4.0. Contexts will be scored as a whole instead of individual statements and only the relevant sentences will be returned. A score of 1 is now returned if a relevant sentence is found, and 0 otherwise."
```

Comment:
```suggestion
    The output of the ContextRelevanceEvaluator will change in Haystack 2.4.0. Contexts will be scored as a whole instead of individual statements and only the relevant sentences will be returned. A score of 1 is now returned if a relevant sentence is found, and 0 otherwise.
```